common/common.manylinux_2_28

```diff
@@ -21,7 +21,7 @@ RUN \
   set -x && \
   # Remove sudo provided by devtoolset since it doesn't work with
   # our sudo wrapper calling gosu.
-  rm -f /opt/rh/gcc-toolset-11/root/usr/bin/sudo && \
+  rm -f /opt/rh/gcc-toolset-12/root/usr/bin/sudo && \
```

Comment:
This line was last touched [5 months ago](https://github.com/thewtex/dockcross/commit/ada2c8dad605fda5a81db2243e31371c0b2c2ed3). Why does the problem manifest with an image from 2 days ago, but not the one from beginning of this month?

Reply:
The images is built [from the `latest`](https://github.com/dockcross/dockcross/blob/08e2f6eec17d22cbb0c726145bb9b6cd05fa93ce/manylinux_2_28-x64/Dockerfile.in#L2) manylinux tag. The toolchain in that image changed during that time period.

It seems that they recently started adding date / hash tags to the manylinux images like we do in dockcross. I will add those.